src/langs/en/auth.json

```diff
@@ -11,5 +11,8 @@
   "SIGN_UP_HEADER": "Sign Up",
   "EMAIL_FIELD_TEXT": "Email",
   "SWITCH_ACCOUNT_TEXT": "Switch to another account",
-  "NOT_SIGNED_IN_TOOLTIP": "You are not signed in."
+  "NOT_SIGNED_IN_TOOLTIP": "You are not signed in.",
+  "SIGN_IN_SUCCESS_TITLE": "Your request to be signed has succeeded",
+  "SIGN_IN_SUCCESS_TEXT": "Please check your inbox <bold>{{email}}</bold> to retrieve your personal login link. This can take several minutes",
+  "SIGN_IN_SUCCESS_EMAIL_PROBLEM": "If you did not receive any email, check your spam. If you used an institutional email (ie. school, company), it might have been blocked and you will need more time to release the email."
```

Comment:
```suggestion
  "SIGN_IN_SUCCESS_EMAIL_PROBLEM": "If you did not receive any email, check your spam. If you used an institutional email (ie. school, company), it might have been blocked and you will have to wait until the email is release by your spam system."
```